code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -204,7 +204,8 @@
 	if(M.a_intent == INTENT_DISARM)
 		if(M.buckled_mobs && (src in M.buckled_mobs) && M.riding_datum)
 			M.riding_datum.force_dismount(src)
-	dna.species.spec_attack_hand(M, src)
+	if(istype(M))

```

Comment:
I'd prefer if you get rid of the argument bound variables instead and set it in spec_attack_hand body.